protocol/v2/ssv/validator/committee_queue.go

```diff
@@ -90,59 +98,53 @@ func (v *Committee) ConsumeQueue(logger *zap.Logger, slot phase0.Slot, handler M
 	if err != nil {
 		return err
 	}
+	
+	state := *q.queueState
+	state.Slot = slot
```

Comment:
I still don't get why this is here and not in construction (StartDuty) where the queue is created. (its 0 there). Don't forget to add it in the new HandleMessage where we create the queue if we don't have it yet on a new msg.

Reply:
@y0sher, fixed this. Added setting the `state.Slot` to `StartDuty` and  `HandleMessage`.